dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/OrderAndPaginationExporterTest.java

```diff
@@ -235,6 +239,7 @@ void shouldOrderTrackedEntitiesByCreatedAsc()
             .trackedEntityUids(Set.of("QS6w44flWAf", "dUE514NMOlo"))
             .trackedEntityTypeUid(trackedEntityType.getUid())
             .orderBy("created", SortDirection.ASC)
+            .user(importUser)
```

Comment:
nit: Please leave `orderBy` as the last call before build. That is what this test is about so should be easy to spot 😅